### DISCUSSION TITLE
Does UNO has non-UI api  for StorageAccessPermission?

### DISCUSSION THREAD

**Discussion by DJDQfff** [unoplatform/uno#13103] [Q&A]
The UWP API: Windows.Storage.AccessCache.StorageApplicationPermissions.FutureAccessList 

When I try to use it  in UnoPlatform, vs showed that it is not implemtend in Uno. 

Is there any relative api to change it  or similar solution?

This involves storage permissions, I just find some words about the permission in the article of FilePicker.

**Comment by jeromelaban** [comment]
@MartinZikmund may have some input there!

**Comment by MartinZikmund** [comment]
@DJDQfff Not yet unfortunately, but if it is something you would like us to prioritize, please add a thumbs up on this issue - https://github.com/unoplatform/uno/issues/5343
